zebra-state/src/service/finalized_state/disk_format/tests/snapshots/block_header_by_height_raw_data@mainnet_0.snap

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Can we delete the unused snapshot files as well?

If you re-run the tests with:
```sh
cargo insta test --review --delete-unreferenced-snapshots
```

That should delete the old snapshots that we aren't using any more, and turn the git diff into a move.

Reply:
I run the command, and it deleted unused snapshots. I committed the deletions at d99d7bfb4a507cff9fc808c8871835e276f1a01f. There were no moves though.

Reply:
In the full PR diff, git shows those files as moved, because they are very similar to the deleted & added files.